envs/example/standard/hosts

```diff
@@ -10,9 +10,16 @@ controller2
 controller1
 controller2
 
+[mongo_db]
+controller1
+controller2
+
 [db_arbiter]
 compute1
 
+[mongo_arbiter]
```

Comment:
Initially that's how I had it, but Paul suggested that I separate it out so MongoDB is not tightly coupled with MySQL considering it is currently an optional service with Ceilometer. This also allows for future architectural flexibility as we scale. Let me know which way is better for now.